### PR TITLE
Add git autoupdate config in wavesurfer.js

### PR DESCRIPTION
**Pull request by Piicksarn**

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Let's keep its structure as what is looks like right now, set basePath to dist.

**Comment by Piicksarn** [comment]
How about this?

File: ajax/libs/wavesurfer.js/package.json [hunk 1]
```diff
@@ -10,5 +10,15 @@
...
   "license": "CC-BY 3.0",
   "bugs": "https://github.com/katspaugh/wavesurfer.js/issues",
-  "homepage": "https://github.com/katspaugh/wavesurfer.js"
+  "homepage": "https://github.com/katspaugh/wavesurfer.js",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/katspaugh/wavesurfer.js.git",
+    "basePath": "dist",
+    "files": [
+      "../wavesurfer.js",
```
Comment (PeterDaveHello) [inline ajax/libs/wavesurfer.js/package.json]: Should not use `../` here.

**Comment by PeterDaveHello** [comment]
Please also add `plugin` folder.

**Comment by Piicksarn** [comment]
thanks for remind me the details, and I correct it.

**Comment by PeterDaveHello** [comment]
Should rebase with master.

**Comment by Piicksarn** [comment]
i've already rebase it with master~

**Pull request merged**